utils/zsh/Makefile

```diff
@@ -42,13 +42,27 @@ endef
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--bindir=/bin \
```

Comment:
Actually, it is /bin, but not a `$PREFIX/bin`, and `--libdir` is not `--libexecdir`, which is defined there.
But, anyway, I'm changing this already.